src/Middleware/Rewrite/ref/Microsoft.AspNetCore.Rewrite.csproj

```diff
@@ -7,6 +7,7 @@
     <Compile Include="Microsoft.AspNetCore.Rewrite.netcoreapp3.0.cs" />
     <Reference Include="Microsoft.AspNetCore.Hosting.Abstractions"  />
     <Reference Include="Microsoft.AspNetCore.Http.Extensions"  />
+    <Reference Include="Microsoft.AspNetCore.Server.IIS"  />
```

Comment:
Hrm, it's unfortunate that we need to reference `.Server.IIS` just to get `IServerVariablesFeature`. Could we consider relocating that feature interface? It's very simple.

Reply:
I can go either way on this change. In one hand, the ServerVariable feature is pretty much purely an IIS feature, so it makes sense to live in that namespace. On the other hand, the feature is simple. 

I don't think it's a huge deal having Rewrite depend on IIS.

Reply:
This your call. Would we relocate the interface to a common project and then have both `.Server.IIS` and `Rewrite` depend on it?

I'll leave it this way for now, but feel free to drop another comment here if you decide you don't want the current dependency.

Reply:
How old is the feature? Moving it is likely breaking.

The dependencies aren't really a problem anymore for anything in the shared framework.

Reply:
I agree, I don't think the dependency issue is blocking. I just wanted to raise it. The shared framework does make this point mostly moot. There's no scenario where Rewrite is available and Server.IIS isn't.

Ok, let's go ahead then.

Reply:
Lets move it and type forward.

Reply:
It doesn't need to block this PR tho

Reply:
@davidfowl we're going to have trouble expanding the features model if we insist that every feature interface must go into a central dll. Also consider that feature interfaces can bring with them additional types specific to their scenario (e.g. IResponseCookiesFeature, IResponseCookies, CookieOptions).  Cases like this where the feature is component specific warrant locating the feature interface closer to that component. What about adding new packages like IIS.Abstractions? That gives you some separation but still give us a growth model. 

A more extreme example would be if a component that wasn't in the shared runtime wanted to expose a feature interface. It wouldn't make sense to add a feature to Http.Features for a component we wouldn't even bundle with the runtime.

Reply:
I understand the concern and I share it. I don't think all features should go into there (for e.g. I'm not suggesting we move all of the kestrel internal features) but server variables are generic enough to imagine supporting them outside of IIS and dependency free enough that I think it's reasonable to move it to the features assembly. 

We don't have ANIs anymore...